gcp/live/dev/k8s/cluster/terraform.tfvars

```diff
@@ -19,4 +19,7 @@ terragrunt = {
 
 # ↓ Module configuration (empty means all default)
 
-node_type = "n1-standard-2"
+node_type          = "n1-standard-2"
+initial_node_count = 2
```

Comment:
I don't understand why this is 2 in dev, but 1 in stg and prd.

Reply:
Is 2 enough nodes to have room for everything? I remember we had to add a fourth zone to facilitate rolling-updates and the like.

Reply:
We need at least 4 `n1-standard-2` nodes to have room for everything.

There are 4 zones in us-central1 (`additional_zones   = ["us-central1-a", "us-central1-b", "us-central1-c", "us-central1-f"]`), with `initial_node_count = 1` it makes 4 worker nodes total.

But there are only 3 zones in us-east1 (b,c,d), so to have 4 worker nodes I changed initial node count to 2 and used only us-east1-b and us-east1-c as additional zones.